gap/directories.gi

```diff
@@ -56,6 +53,21 @@ end);
 InstallGlobalFunction(PKGMAN_InsertPackageDirectory,
 function(pkgpath)
   local parent;
+
+  if IsBound(ExtendPackageDirectories) then
+    # GAP 4.14 or newer
```

Comment:
this is only introduced in GAP 4.15. @mtorpey maybe you could edit the comment here and the one below the next time you touch these files?